components/public-api/gitpod/v1/prebuild.proto

```diff
@@ -60,6 +63,20 @@ message WatchPrebuildResponse {
   Prebuild prebuild = 1;
 }
 
+message ListOrganizationPrebuildsRequest {
+  message Filter {
+    PrebuildPhase.Phase status = 1;
```

Comment:
What do you think about adding a `searchTerm` property here since we plan on needing it?